editor/SCsub

```diff
@@ -81,7 +81,6 @@ if env['tools']:
     env.add_source_files(env.editor_sources, "*.cpp")
     env_thirdparty = env.Clone()
     env_thirdparty.disable_warnings()
-    env_thirdparty.add_source_files(env.editor_sources, ["#thirdparty/misc/clipper.cpp"])
```

Comment:
The whole `env_thirdparty` is unused now.

Reply:
Hmm, well then it's strange why it compiles without linking errors.

Reply:
Well I mean it does nothing, you remove the only file that was built within `env_thirdparty` which is created two lines above. So you should remove it.